Failed to install acorn-node@1.6.1

`npm install` error message:
```
...
npm ERR! node v6.12.0
npm ERR! npm  v2.15.12
npm ERR! code EPEERINVALID

npm ERR! peerinvalid The package acorn@5.7.3 does not satisfy its siblings' peerDependencies requirements!
...
```
It seems that updated dependency acorn-dynamic-import@4.0.0 requires acorn@6.0.2
But acorn-node@1.6.1 provides  acorn@^6.0.1

```
...
│ │ │   ├── https-browserify@1.0.0
│ │ │   ├─┬ insert-module-globals@7.2.0
│ │ │   │ ├─┬ acorn-node@1.6.1
│ │ │   │ │ ├── UNMET PEER DEPENDENCY acorn@6.0.2
│ │ │   │ │ ├── acorn-dynamic-import@4.0.0
│ │ │   │ │ └── acorn-walk@6.1.0
│ │ │   │ ├─┬ concat-stream@1.6.2
│ │ │   │ │ └── buffer-from@1.1.1
│ │ │   │ ├── is-buffer@1.1.6
...
```

--- a/package.json
+++ b/package.json
@@ -7,18 +7,18 @@
     "url": "https://github.com/browserify/acorn-node/issues"
   },
   "dependencies": {
-    "acorn": "^6.0.1",
+    "acorn": "^6.0.2",
     "acorn-dynamic-import": "^4.0.0",
-    "acorn-walk": "^6.0.1",
+    "acorn-walk": "^6.1.0",
     "xtend": "^4.0.1"
   },
   "devDependencies": {
-    "acorn-bigint": "^0.3.0",
+    "acorn-bigint": "^0.3.1",
     "acorn-import-meta": "^0.3.0",
-    "buble": "^0.19.3",
+    "buble": "^0.19.6",
     "mkdirp": "^0.5.1",
     "standard": "^11.0.1",
-    "tape": "^4.8.0"
+    "tape": "^4.9.1"
   },
   "homepage": "https://github.com/browserify/acorn-node",
   "keywords": [

